packages/core/admin/ee/admin/src/pages/SettingsPage/pages/ReviewWorkflows/hooks/useReviewWorkflows.ts

```diff
@@ -41,7 +45,7 @@ export function useReviewWorkflows(params = {}) {
   return {
     // meta contains e.g. the total of all workflows. we can not use
     // the pagination object here, because the list is not paginated.
-    meta: React.useMemo(() => data?.meta ?? {}, [data?.meta]),
+    meta: React.useMemo(() => data?.meta ?? {}, [data?.meta]) as GetAll.Response['meta'],
```

Comment:
How come this needs casting? Would it be better to return null so we know if it's truthy it's our object? 🤔